packages/syft/setup.cfg

```diff
@@ -111,7 +114,6 @@ telemetry =
     opentelemetry-instrumentation==0.48b0
     opentelemetry-instrumentation-requests==0.48b0
     opentelemetry-instrumentation-fastapi==0.48b0
-    opentelemetry-instrumentation-pymongo==0.48b0
```

Comment:
We need to add `opentelemetry-instrumentation-psycopg2==0.48b0`

Reply:
Resolved by: https://github.com/OpenMined/PySyft/pull/9176/commits/7a6458883145d1efd971513921cb83a4e4ca8114